frame/ethereum/src/tests/eip7702.rs

```diff
@@ -24,6 +24,22 @@ use ethereum::{AuthorizationListItem, TransactionAction};
 use pallet_evm::{config_preludes::ChainId, AddressMapping};
 use sp_core::{H160, H256, U256};
 
+// Ultra simple contract that just returns 42 for any call
+// This is pure runtime bytecode that:
+// 1. Pushes 42 (0x2a) onto the stack
+// 2. Pushes 0 (memory offset) onto the stack
+// 3. Stores 42 at memory offset 0 (MSTORE)
+// 4. Pushes 32 (return data size) onto the stack
+// 5. Pushes 0 (memory offset) onto the stack
+// 6. Returns 32 bytes from memory offset 0 (RETURN)
+const _SIMPLE_CONTRACT_RUNTIME: &str = "602a60005260206000f3";
+
+// Creation bytecode that deploys the runtime bytecode above
+// This pushes the runtime code to memory and returns it
+const SIMPLE_CONTRACT_CREATION: &str = "69602a60005260206000f3600052600a6016f3";
+
+const EIP7702_DELEGATION_INDICATOR: [u8; 3] = [0xef, 0x01, 0x00];
```

Comment:
We should import `EIP7702_DELEGATION_INDICATOR` from the lib file

Reply:
I had actually defined them in `evm` for this purpose. Thanks for pointing it out 👍 

Addressed in https://github.com/polkadot-evm/frontier/pull/1727/commits/fc7adaf0edaedb1b0591a802dfd339725b7e1a99